### PR TITLE
Added OneTouch functionality. Automatically falls back to token.

### PR DESCRIPTION
**Pull request by jarodreyes**
There are many changes to the application. This branch should not be merged until the new tutorial has gone live on twilio.com/docs.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -6,7 +6,8 @@
 	"type": "project",
 	"require": {
 		"laravel/framework": "5.0.*",
-		"authy/php": "~2.2"
+		"authy/php": "~2.2",
+		"barryvdh/laravel-debugbar": "~1.8"
```
Comment (rickyrobinett) [inline composer.json]: Probably want to pull out the debugbar from the final tutorial.

File: config/app.php [hunk 2]
```diff
@@ -13,7 +13,7 @@
 	|
 	*/
 
-	'debug' => env('APP_DEBUG'),
+	'debug' => true,
```
Comment (rickyrobinett) [inline config/app.php]: I'd rollback this change but you may already be doing that.

File: config/app.php [hunk 3]
```diff
@@ -144,6 +144,7 @@
 		'App\Providers\ConfigServiceProvider',
 		'App\Providers\EventServiceProvider',
 		'App\Providers\RouteServiceProvider',
+		'Barryvdh\Debugbar\ServiceProvider',
```
Comment (rickyrobinett) [inline config/app.php]: remove debugbar references in this file

File: config/database.php [hunk 4]
```diff
@@ -1,11 +1,9 @@
...
-$url = parse_url(getenv("DATABASE_URL"));
-
-$host = $url["host"];
-$username = $url["user"];
-$password = $url["pass"];
-$database = substr($url["path"], 1);
+$host = "localhost";
+$username = "postgres";
+$password = "postgres";
+$database = "2faf";
```
Comment (rickyrobinett) [inline config/database.php]: Something you may already be planning on but I'd rollback this change as well

**Comment by jarodreyes** [comment]
Alright @rickyrobinett Look good?

**Pull request merged**